scanner/src/test/assets/__files/clearlydefined_scancode.json

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
Where does the `__files` convention in the path come from? Should we maybe be more specific and replace it with "wiremock(s)"? Or slightly less specific and call it "response(s)"?

Reply:
This is unfortunately a convention of the WireMock library. You can configure the root path of your test files, but it insists on appending a "__files" subfolder. I have found no way to change this.

Reply:
Good to know. How about add a very brief `README.md` then to the `__files` directory, explaining exactly this WireMock convention? So people just browsing the files do not wonder like I did.